sdks/java/core/src/main/java/org/apache/beam/sdk/io/BigQueryIO.java

```diff
@@ -491,24 +471,25 @@ Bound withTestServices(BigQueryServices testServices) {
 
       @Override
       public void validate(PInput input) {
-        if (validate) {
```

Comment:
Not sure about this change -- looks functional. May be correct but in a separate PR?